webapp/src/components/Navbar.tsx

```diff
@@ -27,14 +27,18 @@ const Navbar = () => {
           </StyledLogo>
         </Link>
       </Grid>
+
       <Grid item xs="auto">
-        <IconButton sx={{ mr: 1 }} href="/">
+        <IconButton sx={{ mr: 1 }} href="/" title="Home">
           <HomeIcon />
         </IconButton>
-        <IconButton sx={{ mr: 1 }} href="/calendar">
+        <IconButton sx={{ mr: 1 }} href="/competencies" title="Competencies">
+          <EngineeringIcon />
+        </IconButton>
+        <IconButton sx={{ mr: 1 }} href="/calendar" title="Calendar">
           <CalendarMonthIcon />
         </IconButton>
-        <IconButton sx={{ mr: 1 }} onClick={openMeetingsDrawer}>
+        <IconButton sx={{ mr: 1 }} onClick={openMeetingsDrawer} title="Drawer">
           <PeopleIcon />
         </IconButton>
         <Button
```

Comment:
I believe @veranika-karpava is referring to this: https://mui.com/material-ui/react-tooltip

Veranika, I can't reproduce your three-second delay, as on my machine the native browser tooltip shows up in Firefox after 25/60th of a second (ask me later how I know), so less than half a second.

But I agree with the point that Veranika makes here anyway, and the page linked above gives a great example of a tooltip being applied to an IconButton in its "Basic Tooltip" section. Normally I'd be hesitant to replace this functionality with an MUI component because it might impact accessibility, but it seems it was implemented in an accessible manner.